bin/client/src/lib.rs

```diff
@@ -116,7 +117,15 @@ where
         l1_provider.clone(),
         l2_provider.clone(),
     );
-    let executor = KonaExecutorConstructor::new(&cfg, l2_provider.clone(), l2_provider, None);
+    let executor = KonaExecutorConstructor::new(
+        &cfg,
+        DefaultEvmConfig::new(
+            // ZTODO: Use custom chainspec here, not defaulting to OP Mainnet
+            Arc::new(OpChainSpecBuilder::optimism_mainnet().build()),
+        ),
```

Comment:
@clabby Is this client only for OP Mainnet (ie is this safe) or should it work for any chain that uses FPVM? If the latter, I'll construct ChainSpec out of data in the RollupConfig.

Reply:
It should work for any chain with a `RollupConfig`, yep!